src/components/directorybrowser/directorybrowser.js

```diff
@@ -1,4 +1,4 @@
-define(['loading', 'dialogHelper', 'dom', 'listViewStyle', 'emby-input', 'emby-button', 'paper-icon-button-light', 'css!./directorybrowser', 'formDialogStyle', 'emby-button'], function(loading, dialogHelper, dom) {
+define(['loading', 'dialogHelper', 'dom', 'listViewStyle', 'emby-input', 'paper-icon-button-light', 'css!./directorybrowser', 'formDialogStyle', 'emby-button'], function(loading, dialogHelper, dom) {
```

Comment:
why `define` uses more string names than a function is accepting parameters?..

Reply:
that array is the dependencies that are required to be loaded and made available to the callback. Even if you dont use them , some of these dependencies may inject some code globally, or load some css into the browser. That's why they probably are required but not used in the callback.